priv/repo/migrations/20200311182142_drop_tfa_enabled_from_users.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Hexpm.RepoBase.Migrations.DropTfaEnabledFromUsers do
+  use Ecto.Migration
+
+  def change do
+    alter table(:users) do
+      remove(:tfa_enabled)
```

Comment:
nitpick: before this lands, could we remove AddTwaEnabledToUsers and DropTfaEnabledFromUsers migrations since they cancel each other out?